src/tropical_maxplus.jl

```diff
@@ -1,12 +1,16 @@
-export Tropical, TropicalF64, TropicalF32, TropicalF16, content
+export Tropical, TropicalF64, TropicalF32, TropicalF16
+export content, neginf, posinf
```

Comment:
It is a better practise to move all export statements to the `Tropical.jl` file to avoid potential repeated export.